packages/charts/chart-web-components/src/horizontal-bar-chart/horizontal-bar-chart.styles.ts

```diff
@@ -164,4 +165,17 @@ export const styles: ElementStyles = css`
     ${typographyBody1StrongStyles}
     color: ${colorNeutralForeground1};
   }
-`;
+`.withBehaviors(
+  forcedColorsStylesheetBehavior(css`
+    .legend-rect,
+    .tooltip-line {
```

Comment:
is this issue not occuring for donut chart?

Reply:
This fix is already in place in donut chart.